environment/src/main/java/com/sequenceiq/environment/proxy/v1/controller/ProxyController.java

```diff
@@ -74,9 +81,17 @@ public ProxyResponse post(ProxyRequest request) {
     }
 
     @Override
-    public ProxyResponse delete(String name) {
+    public ProxyResponse deleteByName(String name) {
+        notify(ResourceEvent.PROXY_CONFIG_DELETED);
```

Comment:
I think we should send notification if the deletion process has been finished successfully :)